etc/kayobe/kolla/config/octavia.conf

```diff
@@ -0,0 +1,11 @@
+[controller_worker]
+# The following currently need to be configured outside of kolla-ansible.
+amp_secgroup_list = ecdb2fc2-336b-414d-b883-64063abc1c0d
+amp_boot_network_list = fa44bf40-60cf-4bfc-87c7-3b6141725a97
+amp_flavor_id = 47ba5ae5-d77c-492d-bc07-ad46ea466091
+
+[service_auth]
+# Upstream should use `project_name = service` instead of
+# openstack_auth.project_name which defaults to admin which means we are
+# required to add octavia user to admin project.
+project_name = service
```

Comment:
Glad to see this worked. https://review.opendev.org/#/c/727160/ proposes it, but still have concerns about upgrades.